singleFrameProcCoreTasks/isrtask.rst

```diff
@@ -0,0 +1,432 @@
+
+#######
+IsrTask 
+#######
+
+Instrument Signature Removal (ISR) is a sequence of steps taken to
+'clean' images of various aspects of defects that any system of optics
+and detectors will imprint on an image by default, and is generally
+one of the very first procedures carried out on an exposure.
+
+In more detail, though the process for correcting imaging data is very
+similar from camera to camera, depending on the image, various of the
+defects will be present and need to be removed, and thus the sequence
+of steps taken will vary from image to image.  Generally these
+corrections are done one CCD at a time, but on all the amplifier
+sub-images at once for a CCD.  
```

Comment:
Ok, i condensed according to the above, and collapsed paragraphs though -- my personal preference is *not* to do that, and i do believe in the age and era of web reading, small short paragraphs is much more readable.  (I believe this is espoused by the EPPO philosophy, too..)